src/vos/vos_obj_cache.c

```diff
@@ -196,7 +196,8 @@ vos_obj_release(struct daos_lru_cache *occ, struct vos_object *obj)
 int
 vos_obj_hold(struct daos_lru_cache *occ, struct vos_container *cont,
 	     daos_unit_oid_t oid, daos_epoch_t epoch,
-	     bool no_create, uint32_t intent, struct vos_object **obj_p)
+	     bool no_create, bool cache_only, uint32_t intent,
+	     struct vos_object **obj_p)
```

Comment:
Instead of adding extra parameter to vos_obj_hold() and change so many calls, I think it's better to introduce another vos_obj_cache_lookup() or something (and make vos_obj_hold() call into this new function), then use the vos_obj_cache_lookup() in vos_agg_obj(), what do you think?